XLS-0070-credentials/README.md

```diff
@@ -341,6 +340,7 @@ _Note: the transaction will succeed if (non-expired) credentials are included, b
 
 * If a credential isn't valid, the transaction fails.
 * If a credential is expired, the credential is deleted.
+* If the transaction is pre-authorized (either via account or via credential), the transaction will succeed.
 
 ## 9. RPC: `deposit_authorized`
```

Comment:
No mentions about ledger_entry changes (additional field authorized_credentials in deposit_preauth parameter)